MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -32,6 +32,7 @@ internal sealed class Actor : MixedRealityExtensionObject, ICommandHandlerContex
         private UnityLight _light;
         private UnityCollider _collider;
         private LookAtComponent _lookAt;
+        private Dictionary<Guid, AudioSource> _soundInstances;
```

Comment:
>Dictionary [](start = 16, length = 10)

Does this mean an actor can have multiple sound instances? I didn't know we had this capacity, nice!